using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -72,7 +72,7 @@ Use caution when manually clearing the queue. This operation cannot be reverted.
 **Syntax**
```

Comment:
We usually use this format: 
```
## apiName
### Swift
** Syntax **
** Example **
```
and we can also remove the API name headers inside the tabs

Reply:
Updated with  recommended format.